source/FrameTimer.cpp

```diff
@@ -14,6 +14,10 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include <thread>
 
+#ifdef _WIN32
+#include <Windows.h>
```

Comment:
```suggestion
#include <windows.h>
```
Not sure if there's a styleguide behind it, but that's how it's done in [other files](https://github.com/endless-sky/endless-sky/blob/master/source/Files.cpp#L23).